modules/API/API.hpp

```diff
@@ -104,8 +105,10 @@ class API : public Everest::ModuleBase {
     // insert your private definitions here
     std::vector<std::thread> api_threads;
     bool running = true;
-    std::vector<std::unique_ptr<SessionInfo>> info;
-    std::vector<json> hw_capabilities_json;
+    // We don't care here about locality since we're never iterating but must
+    // ensure that our iterators don't get invalidated.
```

Comment:
I wouldn't leave this command here, this should be evident for this piece of code (it could also be moved into the commit message)